packages/edit-site/src/components/sidebar-navigation-item/index.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { __experimentalItem as Item } from '@wordpress/components';
+
+export default function SidebarNavigationItem( { className, ...props } ) {
```

Comment:
This component is mostly here for design purposes, to ensure that we can render the menu items in the sidebar properly.

I also noticed that a similar design is also applied to the "NavigationInspector" which relies on the "TreeGrid" component.

It would have been great if all these components would "compose" naturally with the same design without us styling them over and over again.